webpack.config.js

```diff
@@ -0,0 +1,66 @@
+const { parse, resolve } = require('path');
+const { sync: glob } = require('fast-glob');
+const WebpackAssetsManifest = require('webpack-assets-manifest');
+const RailsI18nWebpackPlugin = require('@18f/identity-rails-i18n-webpack-plugin');
+
+const mode = process.env.NODE_ENV || 'development';
+const isProduction = mode === 'production';
+const hashSuffix = isProduction ? '-[contenthash:8].digested' : '';
```

Comment:
TODO: _Maybe_ removed the `.digested` bit here. That was added in anticipation of [upcoming Sprockets detection](https://github.com/rails/sprockets/pull/718), but since we forgo Sprockets in this implementation, we may not care to bother. Then again, it's not hurting anything to keep.

Reply:
> TODO: _Maybe_ remove the `.digested` bit here. That was added in anticipation of [upcoming Sprockets detection](https://github.com/rails/sprockets/pull/718), but since we forgo Sprockets in this implementation, we may not care to bother. Then again, it's not hurting anything to keep.

Removed in 97bccf3.